Fix build (broken because of some images in flavor-manager doc)

```
Error:  Client bundle compiled with errors therefore further build is impossible.
Module not found: Error: Can't resolve './images/flavor-manager/flavors-1.png' in '/home/runner/work/docs/docs/docs/02-iaas/components'
Module not found: Error: Can't resolve './images/flavor-manager/flavors-2.png' in '/home/runner/work/docs/docs/docs/02-iaas/components'
Error: Process completed with exit code 1.
```

Related change: https://github.com/osism/osism.github.io/commit/b87031906827df616885b5ffca00c52e1532006e#diff-75b0b796b0cf9eaba796b5336939e8b0eb5564e10f9a01126e7e99fc7849474dR103-R114

--- a/docs.package.json
+++ b/docs.package.json
@@ -7,13 +7,7 @@
   },
   {
     "repo": "osism/osism.github.io",
-    "source": "docs/guides/operations-guide/openstack/image-manager.md",
-    "target": "docs/02-iaas/components",
-    "label": ""
-  },
-  {
-    "repo": "osism/osism.github.io",
-    "source": "docs/guides/operations-guide/openstack/flavor-manager.md",
+    "source": "docs/guides/operations-guide/openstack/*",
     "target": "docs/02-iaas/components",
     "label": ""
   },

--- a/docs.package.json
+++ b/docs.package.json
@@ -7,7 +7,7 @@
   },
   {
     "repo": "osism/osism.github.io",
-    "source": "docs/guides/operations-guide/openstack/*",
+    "source": "docs/guides/operations-guide/openstack/day2-operations/*",
     "target": "docs/02-iaas/components",
     "label": ""
   },

